industrial_ci/src/tests/source_tests.sh

```diff
@@ -78,6 +78,8 @@ file) # When UPSTREAM_WORKSPACE is file, the dependended packages that need to b
     elif [ -e $TARGET_REPO_PATH/$ROSINSTALL_FILENAME ]; then
         # install (maybe unreleased version) dependencies from source
         $ROSWS merge file://$TARGET_REPO_PATH/$ROSINSTALL_FILENAME
+    else
+        error "UPSTREAM_WORKSPACE file '$TARGET_REPO_PATH/$ROSINSTALL_FILENAME[.$ROS_DISTRO]' does not exist"
```

Comment:
Output of this line looks a bit confusing:

https://travis-ci.org/ros-industrial/industrial_ci/jobs/208837309#L163
```
\e[31mUPSTREAM_WORKSPACE file '/root//home/travis/build/ros-industrial/industrial_ci/.i.do.not.exist[.jade]' does not exist\e0m
```

- There may be a bug in `error` function. The extraneous prefix in the beginning (`\e[31mUPSTREAM_WORKSPACE`).
- What's the purpose for enclosing the file extension as `[.$ROS_DISTRO]`?